refactor(ast): remove defunct `visit_as` + `visit_args` attrs from `#[ast]` macro

overlookmotel opened the pull request:
#4371 replaced `#[visit_as(...)]` and `#[visit_args(...)]` with `#[visit(as(...))]` and `#[visit(args(...))]`. Remove these defunct helper attributes from the `#[ast]` macro.

overlookmotel commented:
* **#4599** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4599?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

rzvxa reviewed (approved):
Oh, sorry I've missed those 😅

overlookmotel committed:
refactor(ast): remove defunct `visit_as` + `visit_args` attrs from `#[ast]` macro (#4599)

#4371 replaced `#[visit_as(...)]` and `#[visit_args(...)]` with `#[visit(as(...))]` and `#[visit(args(...))]`. Remove these defunct helper attributes from the `#[ast]` macro.

overlookmotel commented:
Thanks Rez!
